### PR TITLE
Add values_mut (fixes #36) and shrink_to_fit (fixes #28).

### LINKED ISSUE #28: Add shrink_to_fit
This method is missing.

### LINKED ISSUE #36: Add values_mut

### PR DESCRIPTION
**Pull request by clarfonthey**

### CODE REVIEW

**Comment by clarfonthey** [comment]
@Gankro @apasel422 you two seem to be the ones able to merge this?

**Review by pczarn — APPROVED** [review]

File: src/lib.rs [hunk 1]
```diff
@@ -191,6 +191,22 @@ impl<V> VecMap<V> {
         }
     }
 
+    /// Trims the `VecMap` of any excess capacity.
```
Comment (pczarn) [inline src/lib.rs:194]: don't you think "excess capacity" could also describe `None`s that may remain at the end of the array?

Comment (clarfonthey) [reply src/lib.rs:194]: yes and I will add a fix for this momentarily!

Comment (pczarn) [reply src/lib.rs:194]: great

**Pull request merged**